src/applications/find-va-forms/containers/SearchResults.jsx

```diff
@@ -241,7 +152,6 @@ const mapStateToProps = state => ({
 const mapDispatchToProps = dispatch => ({
   updatePagination: (page, startIndex) =>
     dispatch(updatePaginationAction(page, startIndex)),
-  updateResults: results => dispatch(updateResultsAction(results)),
 });
```

Comment:
Most of this is just removing the logic for sorting through the now-removed table header